App/dfu/screens/FileSectionView.swift

```diff
@@ -90,28 +94,46 @@ struct FileSectionView: View {
     }
     
     private func onFileOpen(opened fileUrl: URL) {
-        do {
-            print(fileUrl)
+        let downloadTask = URLSession.shared.downloadTask(with: fileUrl) {
```

Comment:
It tries to download a file also when a local file is selected. The purpose of this function was to copy the file to a local dir. I'm not sure if this is also needed for downloading scenario, but I think we should extract separate function to download a file and then eventually invoke onFileOpen func.

Reply:
This method works for both local and external file.